drs_downloader_to_getm.wdl

```diff
@@ -167,6 +170,26 @@ task download {
         fi
 
         echo "~{downloader} ${total_time} seconds" > "~{downloader}.txt"
+
+        cat << EOF > upload_data_table_point.py
+        #!/usr/bin/python
+        import os
+        import sys
+        import json
+        from datetime import datetime
+
+        from terra_notebook_utils import table
+
+        timing_data = dict(date=datetime.utcnow().strftime("%Y-%m-%dT%H%M%S.%fZ"))
+        with open("~{downloader}.txt", 'r') as fh:
+            downloader_name, duration_seconds, _ = fh.read().split()
+            timing_data[downloader_name] = duration_seconds
+
+        table.put_row(downloader_name, timing_data, workspace="DRS Localization Testing", workspace_namespace="anvil-stage-demo")
```

Comment:
Hi @DailyDreaming,
Will this approach, and this line in particular, place the result of each downloader of the workflow run in the same row of the data table (as it does now, and which I like) or will each downloader result be in a different row?
I don't know how the row/entity id is determined. I looked at the TNU table.put_row code to try to answer this, yet the resulting behavior wasn't completely clear to me.

Reply:
It will.  I'll try a different approach, since I think we want a try/except then so that the final step always runs and some default value indicates failure.